content/en/docs/collector/scaling.md

```diff
@@ -0,0 +1,376 @@
+data is slower than receiving data. You can try to increase the queue size,
+which will cause the Collector to consume more memory, but it will also give
+some room for the backend to breathe without permanently dropping telemetry
+data. But if you keep increasing the queue capacity and the queue size keeps
+rising at the same proportion, it’s indicative that you might want to look
+outside of the Collector. It’s also important to note that adding more workers
+here would not be helpful: you’ll only be putting more pressure on a system
+already suffering from a high load.
+
+Another sign that the backend might be having problems is an increase in the
+`otelcol_exporter_send_failed_spans` metric: this indicates that sending data to
+the backend failed permanently. Scaling up the Collector will likely only worsen
+the situation when this is consistently happening.
+
+## How to Scale
+
+At this point, we know which parts of our pipeline needs scaling. Regarding
+scaling, we have three types of components: stateless, scrapers, and stateful.
+
+Most Collector components are stateless. Even if they hold some state in memory,
+it isn’t relevant for scaling purposes.
+
+Scrapers, like the Prometheus receiver, are configured to obtain telemetry data
+from external locations. The receiver will then scrape target by target, putting
+data into the pipeline.
+
+Components like the tail sampling processor cannot be easily scaled, as they
+keep some relevant state in memory for their business. Those components require
+some careful consideration before being scaled up.
+
+### Scaling Stateless Collectors
+
+The good news is that most of the time, scaling the Collector is easy, as it’s
+just a matter of adding new replicas and using an off-the-shelf load balancer.
+When gRPC is used to receive the data, we recommend using a load-balancer that
+understands gRPC. Otherwise, clients will always hit the same backing Collector.
+
+You should still consider splitting your collection pipeline with reliability in
+mind. For instance, when your workloads run on Kubernetes, you might want to use
+DaemonSets to have a Collector on the same physical node as your workloads and a
+remote central Collector responsible for pre-processing the data before sending
+the data to the storage. When the number of nodes is low and the number of pods
+is high, Sidecars might make more sense, as you’ll get a better load balancing
+for the gRPC connections among Collector layers without needing a gRPC-specific
+load balancer. Using a Sidecar also makes sense to avoid bringing down a crucial
+component for all pods in a node when one DaemonSet pod fails.
+
+The sidecar pattern consists in adding a container into the workload pod. The
+[OpenTelemetry Operator](/docs/k8s-operator/) can automatically add that for
+you. To accomplish that, you’ll need an OpenTelemetry Collector CR and you’ll
+need to annotate your PodSpec or Pod telling the operator to inject a sidecar:
```

Comment:
Just confirming: the original post has backticks around terms like `PodSpec` and `Pod` -- did you intentionally remove them?

Reply:
No, the copy and paste didn't keep them since I took from the website. I tried to replace everything but I probably missed some. I'll do another proofread and add back any formatting I missed.